js/cfapi.js

```diff
@@ -0,0 +1,28 @@
+/*
+* All data-related Code for America API(CFAPI) work.
+* See https://github.com/codeforamerica/cfapi for an overview
+* See http://codeforamerica.org/api/ for full documentation
+*/
+
+(function($, window) {
+  $.getJSON( "http://codeforamerica.org/api/organizations/Code-for-Denver", function( data ) {
+    var events = [];
+
+    $(data.current_events).each( function() {
+      var mEvent = this, name, time, image, link, linkText, tile;
+      name = "<div class='event-name'>" + mEvent.name + "</div>";
+      time = "<time datetime="+mEvent.start_time+">" + mEvent.start_time + "</time>";
```

Comment:
I see that the date comes back as a string, so we will have to pass it into a `new Date()` object.